recipes/_audit_plugin.rb

```diff
@@ -14,20 +14,25 @@
 audit_plugin_options['server_audit_syslog_priority'] = \
   node['mariadb']['audit_plugin']['server_audit_syslog_priority']
 
-audit_plugin_options['enable'] = '#server_audit_logging = ON'
+audit_plugin_options['server_audit_logging'] = 'ON'
 
```

Comment:
`'ON'` should be replaced with an attribute so it can be configured and not forced - perhaps `node['mariadb']['audit_plugin']['server_audit_logging']`?

Otherwise LGTM 👍